winstore-template.csv

```diff
@@ -0,0 +1,430 @@
+﻿Field,ID,Type (Type),default,en-gb
+Description,2,Text,,"$STR_ABSTRACT$
+
+$STR_OVERVIEW_BUILD_TITLE$
+
+$STR_OVERVIEW_BUILD_TEXT$
+
+$STR_OVERVIEW_TRANSPORT_TITLE$
+
+$STR_OVERVIEW_TRANSPORT_TEXT$
+
+$STR_OVERVIEW_UPGRADE_TITLE$
+
+$STR_OVERVIEW_UPGRADE_TEXT$
+
+$STR_OVERVIEW_EXTEND_TITLE$
+
+$STR_OVERVIEW_EXTEND_TEXT$
+
+$STR_OVERVIEW_LICENSE_TITLE$
+
+$STR_OVERVIEW_LICENSE_TEXT$"
+ReleaseNotes,3,Text,,"OpenTTD 14.1 fixes a multiplayer desync bug and cleans up a few rough edges with the new ship pathfinder and the unbunching feature, plus plenty of other bugfixes. OpenTTD 14.0 introduces over 40 new features, over 500 bug fixes and 200 other changes, including:
+
+- New timekeeping, use real-time units if you want!
+- Improved ship pathfinding
+- Road vehicle unbunching
+- Social Platform integration
+- Survey support
+- GUI improvements, scalable font
+- New Help and Manuals option, improved default settings
+
+Check out our blogs on openttd.org for full details."
```

Comment:
I am a bit slow, but I missed this entry in my first review.

I would very much prefer if we don't have changelogs hardcoded in this repository. Do we actually need to have it like this? As this feels like asking for trouble :D (and another step in the release pipeline). Is there any way around this, or a way to avoid it?

(to be clear, I am not going to keep back this PR because of this; just want to challenge you if we actually want/need this)

Edit: it is also not translated; so maybe just put an URL in there to the changelog, or website? Not sure .. just thinking out loud, mostly.